controller_manager/src/controller_manager.cpp

```diff
@@ -379,14 +379,14 @@ controller_interface::return_type ControllerManager::switch_controller(
           if (strictness == controller_manager_msgs::srv::SwitchController::Request::STRICT) {
             RCLCPP_ERROR(
               get_logger(),
-              "Could not %s controller with name '%s' because no controller with this name exists",
+              "Could not '%s' controller with name '%s' because no controller with this name exists",
```

Comment:
Solves lint error. (I couldn't do maintainer push...)

```suggestion
              "Could not '%s' controller with name '%s' because " + 
              "no controller with this name exists",
```